### PR TITLE
Include webpacker in Gemfile since their rake task are executed in railties tests

### PR DESCRIPTION
**Pull request by guilleiguaran**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -18,6 +18,8 @@ gem "coffee-rails"
 gem "sass-rails"
 gem "turbolinks", "~> 5"
```
Comment (sikachu) [inline Gemfile]: Do we need a blank line here then? Does it go with the group above?

**Review by sikachu — COMMENTED** [review]
Do we want webpacker here or as development dependency btw?

**Review by rafaelfranca — APPROVED** [review]

**Comment by guilleiguaran** [comment]
I need to investigate this further, including webpacker in the Gemfile freeze the railties tests:

https://travis-ci.org/rails/rails/jobs/410570546

File: Gemfile (hunk 1 above)
Comment (guilleiguaran) [reply Gemfile]: Yes, I think this can be in the same group

**Comment by kaspth** [comment]
Yeah, I couldn’t figure out why the tests froze when I tried this locally too.

**Comment by sikachu** [comment]
So, I did some digging. When I terminated the test, I noticed that there're these lines in the stack trace:

```
Interrupt: 
Traceback (most recent call last):
	7: from test/application/asset_debugging_test.rb:3:in `<main>'
	6: from [home]/.rvm/rubies/ruby-2.5.1/lib/ruby/site_ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
	5: from [home]/.rvm/rubies/ruby-2.5.1/lib/ruby/site_ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
	4: from [home]/Projects/rails/railties/test/isolation/abstract_unit.rb:443:in `<top (required)>'
	3: from [home]/Projects/rails/railties/test/isolation/abstract_unit.rb:443:in `new'
	2: from [home]/Projects/rails/railties/test/isolation/abstract_unit.rb:443:in `initialize'
	1: from [home]/Projects/rails/railties/test/isolation/abstract_unit.rb:450:in `block in <top (required)>'
[home]/Projects/rails/railties/test/isolation/abstract_unit.rb:450:in ``': Interrupt
[home]/.rvm/gems/ruby-2.5.1/bundler/gems/thor-006832ea3248/lib/thor/line_editor/readline.rb:20:in `readline'
[home]/.rvm/gems/ruby-2.5.1/bundler/gems/thor-006832ea3248/lib/thor/line_editor/readline.rb:20:in `readline'
[home]/.rvm/gems/ruby-2.5.1/bundler/gems/thor-006832ea3248/lib/thor/line_editor.rb:7:in `readline'
[home]/.rvm/gems/ruby-2.5.1/bundler/gems/thor-006832ea3248/lib/thor/shell/basic.rb:413:in `ask_simply'
[home]/.rvm/gems/ruby-2.5.1/bundler/gems/thor-006832ea3248/lib/thor/shell/basic.rb:79:in `ask'
...
```

So, it was waiting for user to input something! I then modified `railties/test/isolation/abstract_unit.rb:450` to use `system` instead of backticks so I can get the output.

The last few lines of the output was:

```
Creating JavaScript app source directory
       exist  app/javascript
    conflict  app/javascript/packs/application.js
Overwrite /private/var/folders/fj/w352b41j1ps7y7thz0wnf__c0000gp/T/app_template/app/javascript/packs/application.js? (enter "h" for help) [Ynaqdh] 
```

if I `d` for diff, I got:

```diff
--- /private/var/folders/fj/w352b41j1ps7y7thz0wnf__c0000gp/T/app_template/app/javascript/packs/application.js	2018-08-13 07:53:08.000000000 +0900
+++ /private/var/folders/fj/w352b41j1ps7y7thz0wnf__c0000gp/T/app_template/app/javascript/packs/application.js20180813-48642-1c11lvw	2018-08-13 07:56:11.000000000 +0900
@@ -1,15 +1,10 @@
+/* eslint no-console:0 */
 // This file is automatically compiled by Webpack, along with any other files
 // present in this directory. You're encouraged to place your actual application logic in
 // a relevant structure within app/javascript and only use these pack files to reference
 // that code so it'll be compiled.
+//
+// To reference this file, add <%= javascript_pack_tag 'application' %> to the appropriate
+// layout file, like app/views/layouts/application.html.erb
 
-import Rails from "rails-ujs"
-Rails.start()
-
-import Turbolinks from "turbolinks"
-Turbolinks.start()
-
-import ActiveStorage from "activestorage"
-ActiveStorage.start()
-
-import "channels"
+console.log('Hello World from Webpacker')
Retrying...
Overwrite /private/var/folders/fj/w352b41j1ps7y7thz0wnf__c0000gp/T/app_template/app/javascript/packs/application.js? (enter "h" for help) [Ynaqdh] 
```

I think we need to fix that, as it seems like webpacker override Rails' `application.js` and it breaks `rails new`.

**Comment by guilleiguaran** [comment]
@sikachu thanks for investigating this!!

I guess an acceptable solution is to skip the generation of `app/javascript/packs/application.js` in webpacker if it exists already.

**Comment by lsylvester** [comment]
@guilleiguaran The issue with webpacker overwriting the the application.js seems to be resolved in rails/webpacker@8ca372dc500d16a2343aaade7646647aac6925b3 so it you update this to point at `gem "webpacker", github: "rails/webpacker"` then it should resolve this current point of hanging.  

For me, I am now getting it hanging when it can't find versions of the the rails packages for 6.0.0.alpha.

**Comment by kaspth** [comment]
Let's continue on https://github.com/rails/rails/pull/33622 😊

**Pull request closed without merging**